ESSArch_Core/storage/tests/test_models.py

```diff
@@ -0,0 +1,476 @@
+import os
+import shutil
+import tempfile
+import uuid
+from unittest import mock
+from django.test import TestCase
+
+from ESSArch_Core.configuration.models import Parameter
+from ESSArch_Core.ip.models import InformationPackage
+from ESSArch_Core.storage.models import (
+    DISK,
+    TAPE,
+    CAS,
+    get_storage_type_from_medium_type,
+    medium_type_CHOICES,
+    StorageTarget,
+    StorageMedium, TapeSlot, Robot, StorageObject)
+from ESSArch_Core.util import normalize_path
+
+
+class GetStorageTypeFromMediumTypeTests(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(GetStorageTypeFromMediumTypeTests, cls).setUpClass()
```

Comment:
Python 3 allows `super` to be called without arguments
```suggestion
        super().setUpClass()
```